scripts/sync-kube-mixin.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# make a temporary dir to work in
+TMPDIR=$(mktemp -d -t 'tmp.XXXXXXXXXX')
+trap 'rm -f $TMPDIR' EXIT
+MIXIN_REPO="git@github.com:giantswarm/giantswarm-kubernetes-mixin.git"
+# clone a branch of tag if provided
+BRANCH=${1:-""}
+
+if [ -z "${BRANCH}" ]; then
```

Comment:
I'm not a big fan of curly braces when not needed. I don't think it improves readability, it's just extra characters.
But it's my personal opinion / nitpicking ;)

Reply:
I LIKE double braces in if conditions, I can distinguish that it's an if statement :D

Reply:
You suggest omitting curly braces `{ }` from each variable?

Reply:
Yes, when not needed. Just a suggestion, nothing critical there: we have no coding style guidelines for bash ;)